cSpell.json

```diff
@@ -0,0 +1,102 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [
```

Comment:
ℹ️ Quick note: `words` here will pop up as spelling suggestions in IDEs; `ignoreWords` will not.